event-log/src/main/scala/io/renku/eventlog/subscriptions/package.scala

```diff
@@ -35,4 +36,19 @@ package object subscriptions {
   trait SubscriptionCategoryPayload {
     def subscriberUrl: SubscriberUrl
   }
+
+  sealed trait SubscriptionResult
```

Comment:
I guess as a good practice the subtypes of the `SubscriptionResult` should be nested in the `SubscriptionResult` companion. And, what do you think if the type goes to companion of the `SubscriptionCategory`?

Reply:
So SubscriptionResult I will put in SubscriptionCategoryRegistry as this is the type returned to the endpoint. And I'll put RegistrationResult in the SubscriptionCategory